mm2src/coins/lp_coins.rs

```diff
@@ -3672,7 +3662,7 @@ impl CoinsContext {
                 scan_addresses_manager: ScanAddressesTaskManager::new_shared(),
                 withdraw_task_manager: WithdrawTaskManager::new_shared(),
                 #[cfg(target_arch = "wasm32")]
-                tx_history_db: ConstructibleDb::new(ctx).into_shared(),
+                tx_history_db: ConstructibleDb::new(ctx, None).into_shared(),
```

Comment:
When we connect an external wallet to the Komodo Wallet and then restart the Komodo Wallet with a different account/seed, data conflict in the transaction history can happen due to this line. But this issue only affects protocols that use this implementation (Tendermint does not use this). Therefore, this is a potential bug for other protocols that may use external wallets in the future (e.g., Ethereum with MetaMask would hit this problem).

It's worth to add a TODO here, and I think we should also create an issue right after merging this PR.

Reply:
not really, `db_id` will be ignored if a diff id is provided when `tx_history_db` initialization is called through this line 
https://github.com/KomodoPlatform/komodo-defi-framework/blob/6a0d0b6af9828c608a69e691cd1bb3fdc7a3168d/mm2src/coins/lp_coins.rs#L3798-L3802

Reply:
Missed that part, thanks!